force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -29,7 +29,15 @@ public with sharing class FlowControlLoopsCollections {
      * @return "Hello World!" if x greater than y, otherwise return null.
      */
     public static String helloWorld(Integer x, Integer y) {
-        return null; // Replace null with the variable you used to store the result
+        if(x>y){
+            String output = 'Hello World!';
+            system.debug(output);
+            return output;
+        }else{
+            system.debug('If x<y and x= null and y=null please return null');
+            return null;
```

Comment:
1. See my comment in Slack about the importance of removing debug statements. 
2. On line 33 there is no real need to assign to a variable. All you are doing with that data is returning it. So just `return 'Hello World!`